demo/index.html

```diff
@@ -116,7 +116,7 @@ <h1>three-pathfinding</h1>
 
 			let camera, scene, renderer, controls, raycaster;
 
-			let player, target, level;
+			let player, target, level, levelNav;
```

Comment:
Could you name this `navmesh` to differentiate from the pathfinding module's zone/group objects?